File: sandbox/php/src/TemplateDeleteExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
$config->setUsername("YOUR_API_KEY");
// $config->setAccessToken("YOUR_ACCESS_TOKEN");

try {
    (new Dropbox\Sign\Api\TemplateApi(config: $config))->templateDelete(
        template_id: "f57db65d3f933b5316d398057a36176831451a35",
    );
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling TemplateApi#templateDelete: {$e->getMessage()}";
}

File: sandbox/python/src/TemplateDeleteExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
    # access_token="YOUR_ACCESS_TOKEN",
)

with ApiClient(configuration) as api_client:
    try:
        api.TemplateApi(api_client).template_delete(
            template_id="f57db65d3f933b5316d398057a36176831451a35",
        )
    except ApiException as e:
        print("Exception when calling TemplateApi#template_delete: %s\n" % e)
